README.md

```diff
@@ -12,12 +12,20 @@ Blocking access to the internet can help:
 NetGuard is possibly the first free and open source no-root firewall for Android.
 
 Downloads:
+--------
+
+<a href="https://github.com/M66B/NetGuard/releases"><img src="https://assets-cdn.github.com/images/modules/logos_page/GitHub-Logo.png" height="32" ></a> (stable)
+
+[![Google Play](http://developer.android.com/images/brand/en_generic_rgb_wo_60.png)](https://play.google.com/store/apps/details?id=eu.faircode.netguard&referrer=utm_source%3Dgithub) (stable)  
```

Comment:
I hoped it's useful ( point 2. ): https://developers.google.com/analytics/devguides/collection/android/v4/campaigns#google-play-campaigns